app/livechat/server/lib/Livechat.js

```diff
@@ -268,7 +271,7 @@ export const Livechat = {
 			];
 		}
 
-		if (email && email.trim() !== '') {
+		if (email) {
```

Comment:
What about we benefit from [Object Spread Syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) and all values into `updateUser` object once? 
IMO this improvement will make it easier to understand the code.

Reply:
I left the 2 special cases (email, department) out of the spread, since they include more code than just checking for its existance.